environment_ci.yml

```diff
@@ -8,3 +8,6 @@ dependencies:
   - wheel==0.29.0
   - coverage==4.0.3
   - python-coveralls==2.7.0
+  - dask==0.13.0
+  - numpy==1.11.3
+  - scipy==0.18.1
```

Comment:
These were the latest versions that I could find that had both Python 3.4 and 3.6 builds on conda-forge. Anything newer didn't have Python 3.4 support. So this is as good as we can do until we drop Python 3.4. Alternatively if we do some restructuring of how we handle CIs here, we might be able to bump these. However not planning to address either of these anytime soon.